awx/ui_next/src/components/Lookup/ProjectLookup.jsx

```diff
@@ -116,6 +130,8 @@ function ProjectLookup({
                 key: 'name',
               },
             ]}
+            toolbarSearchableKeys={searchableKeys}
+            toolbarRelatedSearchableKeys={relatedSearchableKeys}
```

Comment:
The OptionsList accepts props `searchableKeys` and `relatedSearchableKeys`, not `toolbar---`. Would you rather rename the props in the OptionsList or update them in the Lookups?